libs/wingsdk/src/target-tf-azure/function.ts

```diff
@@ -283,9 +283,13 @@ export class Function extends cloud.Function {
     lines.push('"use strict";');
     lines.push("module.exports = async function(context, req) {");
     lines.push(
-      `  const body = await (${inflightClient}).handle(context.req.body ?? "");`
+      `try {  
+        const body = await (${inflightClient}).handle(context.req.body);
+        context.res = { body };
+     } catch (error) {
+        context.res = { body: error.message, status: 500 };
+    }`
```

Comment:
Test for this?

Reply:
I tested it manually several times, and the sdk-spec tests working, and logging errors can be some sort of an indirect test (I am not sure how to test it since it's a string, I'll check if there is any library that can simulate azure functions well enough to run this code on them)